### DISCUSSION TITLE
Welcome to Rafiki Discussions!

### DISCUSSION THREAD

**Discussion by cairin** [interledger/rafiki#1] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by vladimirfomene** [comment]
Hello everyone, I'm Vladimir Fomene, a software developer currently located in Accra, Ghana. I came across Interledger in 2020 when I was learning about Cryptocurrencies. Back then, I was looking for something that could solve payments and when I read the pitch for Interledger, I completely got hooked.  I'm here to contribute to the discussions and the developments. Thanks for creating the contribution guidelines @cairin !

**Reply by totorogendut** [reply]
Hi there. That's pretty much how I found about Interledger too! Although, in my case, I'm more involved with Web Monetization API than Interledger ecosystem.

**Comment by A-Mozeak** [comment]
Hi everyone 👋🏾 my name is Alex Mozeak and I am one half of A* Digital LLC, a software R&D studio based in New York. 

Last year we received a grant through Grant for the Web to work on Gib, a solution that can provide gift cards for content that has been monetized using the proposed Web Monetization Standard ([check out our grant report here for more information](https://community.webmonetization.org/amozeak/gib-final-grant-report-35dg)). 

Along the way, we realized that some of the infrastructure that we need in order to act as a payment handler (e.g. for hosting gift card wallets or transferring money) has either not been laid out yet or would require much more experience with Interledger for us to piece together. Thankfully, the Grant for the Web committee introduced us to Rafiki, which we believe will be massively helpful to us in building Gib.

To that end, we would like to contribute to Rafiki's development as much as possible while attempting to build Gib around it. So, how can we help?
